src/locales/pl/location/city_name.ts

```diff
@@ -689,7 +689,7 @@ export default [
   'Sośnicowice',
   'Stalowa Wola',
   'Starachowice',
-  'Stargard Szczeciński',
+  'Stargard,
```

Comment:
Looks like you erased one character too much 😉 

```suggestion
  'Stargard',
```

Reply:
Ouch, sorry! Thanks @ST-DDT.